### PR TITLE
Updates a few details in Contributing.md

### PR DESCRIPTION
**Pull request by gledis69**
* Adds installation of `mitmproxy`. I was getting this error from running regression tests:
	```
	Can't exec "mitmdump": No such file or directory at /home/glediszeneli/citus/src/test/regress/pg_regress_multi.pl line 215.
	```
* Add a comment to alternatively use `install-all` in the setup. Without `install-all` the `mutli-extension` regression test fails.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (gledis69) [reply CONTRIBUTING.md]: I added install-all for Mac by mistake here, but I actually tried setting up in Ubuntu Linux. 
Added the relevant comment for install-all this time in Ubuntu Linux.
Should I add the same comment for the other OS-es as well?

**Review by agedemenli — APPROVED** [review]
LGTM. Please have one more approval before merging this.
Thank you!

**Review by onurctirtir — APPROVED** [review]

**Pull request merged**
